source/Program.cs

```diff
@@ -11,19 +14,35 @@ static void Main(string[] args)
             Console.WriteLine();
             Console.WriteLine($" GitHubSearch version {ProductVersion}");
 
-            if (args.Length <= 0)
+            Options options = GetCommandLineOptions(args);
+
+            PerformSearch(options);
+        }
+
+        private static Options GetCommandLineOptions(string[] args)
+        {
+            var options = new Options();
+            if (Parser.Default.ParseArgumentsStrict(args, options))
             {
-                Console.WriteLine();
-                Console.WriteLine(@" Usage: GitHubSearch.exe ""<searchtoken>""");
-                Environment.Exit(1);
+                if (!options.IsValid)
+                {
+                    Console.WriteLine();
+                    Console.WriteLine(@" Usage: GitHubSearch.exe ""<searchtoken>"" [options]");

```

Comment:
The options would be displayed when using the _-h_ option. I have made some small changes to the `Program` and `Options` classes so it will now always list the options when the arguments are invalid.